### PR TITLE
add dualize MPB function, update tests

### PR DESCRIPTION
**Pull request by chriscoey**
prompted by request from MOSEK to dualize a bunch of PSD constrained problems

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -9,9 +9,9 @@ dat = readcbfdata("example4.cbf")
 
 c, A, b, con_cones, var_cones, vartypes, dat.sense, dat.objoffset = cbftompb(dat)
 
-@test_approx_eq c [1.0, 0.64]
-@test_approx_eq A [-50.0 -31; -3.0 2.0]
-@test_approx_eq b [-250.0, 4.0]
+@test c ≈ [1.0, 0.64]
```
Comment (mlubin) [inline test/runtests.jl:12]: This only works on 0.6. You can use `@test isapprox(...)` instead.

**Review by mlubin — CHANGES_REQUESTED** [review]

File: test/runtests.jl (hunk 1 above)
Comment (chriscoey) [reply test/runtests.jl:12]: It works on 0.6+ right? Are people who would use CBU still using 0.5? We aren't for the paper.

Comment (mlubin) [reply test/runtests.jl:12]: Yes, you can update the package to 0.6+ if you'd like. Please keep the commits separate.

Comment (chriscoey) [reply test/runtests.jl:12]: is that OK?

File: .travis.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@ os:
 #  - osx # disabled until queues are reasonable
 julia:
   - 0.6
-#  - nightly
+  - nightly
```
Comment (mlubin) [inline .travis.yml]: nightly is known to fail. disable it and we'll get tests passing

**Comment by chriscoey** [comment]
okey doke no more nightly

**Review by mlubin — APPROVED** [review]

**Pull request merged**